src/bitmessagemain.py

```diff
@@ -1,49 +1,55 @@
 #!/usr/bin/python2.7
-# Copyright (c) 2012-2016 Jonathan Warren
-# Copyright (c) 2012-2018 The Bitmessage developers
-# Distributed under the MIT/X11 software license. See the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# pylint: disable=no-self-use,too-many-branches,too-many-statements,too-many-locals
+"""
+bitmessagemain.py
+=================
 
-# Right now, PyBitmessage only support connecting to stream 1. It doesn't
-# yet contain logic to expand into further streams.
+Copyright (c) 2012-2016 Jonathan Warren
+Copyright (c) 2012-2018 The Bitmessage developers
+Distributed under the MIT/X11 software license. See the accompanying
+file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-# The software version variable is now held in shared.py
+Right now, PyBitmessage only support connecting to stream 1. It doesn't
+yet contain logic to expand into further streams.
 
+The software version variable is now held in shared.py
+
+"""
+
+from __future__ import absolute_import
+
+import ctypes
+import errno
+import getopt
 import os
+import signal
+import socket
 import sys
+import threading
+from random import randint
+from struct import pack
+from subprocess import call
+from time import sleep
 
 app_dir = os.path.dirname(os.path.abspath(__file__))
 os.chdir(app_dir)
 sys.path.insert(0, app_dir)
 
-
-import depends
-depends.check_dependencies()
```

Comment:
Did you test this?

Reply:
Yes. The import has moved up and the function call moved down.